src/router/index.ts

```diff
@@ -13,14 +17,30 @@ const routes: RouteConfig[] = [
   {
     path: '/about',
     name: 'About',
-    // route level code-splitting
-    // this generates a separate chunk (about.[hash].js) for this route
-    // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue'),
+    component: About,
+    children: [
+
+    ],
+  },
+  {
+    path: '/project',
+    name: 'Project',
```

Comment:
Since we've already had the enum representing the routes, here we would do `name: AppRoute.Project`. Multiple instances